docker/compose/docker-compose.test.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.7'
```

Comment:
This one is not needed anymore I think

Reply:
yeah i didn't clean this up yet, just rebased